script.js

```diff
@@ -4,10 +4,71 @@ openButton.onclick = () => {
+    link: 'link_to_live_version.com',
+  },
+
+  dataDash: {
+    title: 'Data Dashboard Healthcare',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    image: './Normal_Button/data-dashboard-hovered.png',
+    paragraph: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    link: 'link_to_live_version.com',
+  },
+
+  webPort: {
+    title: 'Website Portfolio',
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    image: './Normal_Button/web-portfolio-background.png',
+    paragraph: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    link: 'link_to_live_version.com',
+  },
+};
+
+function showNewSection(projectId) {
+  const container = document.getElementById('yoga-project');
+  const newSection = document.createElement('div');
+  newSection.id = 'pop-up-window';
+  let liItems = `<li><p class='new-section-technology'>${projects[projectId].technologies[0]}</p></li><li><p class='new-section-technology'>${projects[projectId].technologies[1]}</p></li><li><p class='new-section-technology'>${projects[projectId].technologies[2]}</p></li>`;
+  if (projects[projectId].technologies.length > 3) {
+    liItems += `<li><p class='new-section-technology'>${projects[projectId].technologies[3]}</p></li>`;
+  }
+  newSection.innerHTML = `<div class='pop-up-container'><h3 style='margin: 0;'>${projects[projectId].title}<button id='closePopUp' class='close-pop-up' onclick='closeNewSection()'><img src='./Normal_Button/Close_button.png' alt='Close menu'></button></h3>
+                          <ul>${liItems}</ul>
+                          <img src='${projects[projectId].image}' alt='${projects[projectId].title}' class='pop-up-image'/>
+                          <div class='text-and-buttons'><p>${projects[projectId].paragraph}</p>
+                          <div class=button-div><button class=see-project1>See Live<img src='./Normal_Button/seeLive-button.svg' alt='See live'></button><button class='see-project1'>See Source<img src='./Normal_Button/seeSource-button.svg' alt='See source'</button></div></div></div>';`;
```

Comment:
- [ ]Great job working on the requested changes but  I highly recommend to add it some margin here as well since in the mobile version I think the buttons is very close to the text above there is no space to have better UI/UX kindly update this 💪🏻🤩💯✨

- your 
![image](https://user-images.githubusercontent.com/58553711/231621456-4a62ee20-e0fc-4a84-b4db-8c0cadd2e0a4.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/231621549-2a451325-816e-4bf0-8ede-abbf748eb4ff.png)

Reply:
I will do that right away. Thank you very much!